README.md

```diff
@@ -1,3 +1,4 @@
+[![Build status](https://ci.novoda.com/buildStatus/icon?job=spritz)](https://ci.novoda.com/job/spritz/lastBuild/console) [![Download](https://api.bintray.com/packages/novoda/maven/spritz/images/download.svg)](https://bintray.com/novoda/maven/spritz/_latestVersion) [![](https://raw.githubusercontent.com/novoda/novoda/master/assets/btn_apache_lisence.png)](LICENSE.txt)
```

Comment:
`btn_apache_lisence` lol we should likely fix this — but more importantly ❌  it should link to `https://github.com/novoda/spritz/blob/master/LICENSE` (notice, no `.txt` extension)